src/components/list-header/list-header.scss

```diff
@@ -0,0 +1,12 @@
+.list-header {
+    background: #1C1080;
+    color: #FFFFFF;
+    display: inline-block;
+    font-family: 'Titillium Web', sans-serif;
+    font-style: normal;
+    font-weight: normal;
```

Comment:
I don't think we need to specify this. They should be already declared on global css.

Reply:
Hmm, they will be in the global css, but I think at this point in the project we don't have them defined.

Reply:
So... should I remove them or keep them?

Reply:
Let's remove them for now, and we will add them to Storybook only later on. In the client apps we already have the them, so we should be good.

Reply:
Roger that. I removed them.